setup.cfg

```diff
@@ -5,3 +5,8 @@ universal = 1
 exclude = urls.py,tasks.py,*migrations*,*bower_components*
 max-line-length = 100
 ignore = W503
+
+[flake8]
+exclude = urls.py,tasks.py,*migrations*,*bower_components*,*tests*
+max-line-length = 250
```

Comment:
@reifred this line length is too long and could potentially lead to maintainability issues, consider changing it to 80 characters or less for better readability.

Reply:
Thanks for the feedback am going to work on it. I had implemented feedback got from you and @Emmanuel-Dominic.
![image](https://user-images.githubusercontent.com/22131999/58473762-1d46ee80-8152-11e9-862d-49feec0eab77.png)